tests_my/testcases/utils/common.py

```diff
@@ -0,0 +1,27 @@
+import string
+import random
+from selenpy.support import browser
+from selenium.webdriver.support.wait import WebDriverWait
+from tests_my.testcases.utils import constants
+
+
+def random_string(stringLength=10):
+    letters = string.ascii_lowercase
+    return '_' + ''.join(random.sample(letters, stringLength))
+
+
+def get_title():
```

Comment:
Should put these method in browser_helper method

Reply:
I updated